src/main/java/com/fontys/api/repositories/TournamentRepository.java

```diff
@@ -3,6 +3,9 @@
 import com.fontys.api.entities.Tournament;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.Optional;
+
 public interface TournamentRepository extends JpaRepository<Tournament, Integer>
 {
+    Optional<Tournament> findByName(String name);
```

Comment:
Shouldn't this method have an implementation somewhere?

Reply:
Nope, let's thank the magic of hibernate

Reply:
We discussed it and I approve.